noxfile.py

```diff
@@ -110,10 +110,17 @@ def lint_setup_py(session):
 def compliance_test(session):
     """Run SQLAlchemy dialect compliance test suite."""
     session.install("pytest")
+    session.install("mock")
     session.install("-e", ".")
     session.run("python", "create_test_database.py")
     session.run("pytest", "-v")
 
+    # Run SQLAlchemy dialect compliance test suite with OpenTelemetry.
+    session.install("opentelemetry-api==1.1.0")
+    session.install("opentelemetry-sdk==1.1.0")
+    session.install("opentelemetry-instrumentation==0.20b0")
+    session.run("pytest", "-v")
```

Comment:
The unit tests are using `unittest`, not `pytest`. This is just running the compliance tests again. You need to use `session.run("py.test", "--quiet", os.path.join("tests"), *session.posargs)` (or something similar). We can consider whether coverage is useful another time.

Additionally, I'd prefer the unit tests to have their own session so they can be run separately. That way if a developer is writing unit tests they don't have to wait for the compliance tests to finish running to get results.

This should be as simple as writing another function, e.g.
```
nox.session(python=DEFAULT_PYTHON_VERSION)
def unit(session):
    """Run unit tests."""
    // TODO: write code for running the unit tests
```

Reply:
Separate into a new `unit` session for unit test.